src/oatpp/core/data/stream/Stream.hpp

```diff
@@ -405,6 +405,7 @@ class IOStream : public InputStream, public OutputStream {
  */
 class ConsistentOutputStream : public OutputStream {
 public:
+  String floatFormat = OATPP_FLOAT_STRING_FORMAT;
```

Comment:
We don't need this since we gonna have additional methods:

```cpp
v_io_size ConsistentOutputStream::writeAsString(v_float32 value, const char* format)
v_io_size ConsistentOutputStream::writeAsString(v_float64 value, const char* format)
```